src/content/loaders/powerPizzaPodcastLoader.ts

```diff
@@ -0,0 +1,107 @@
+import { feedLoader } from '@ascorbic/feed-loader';
+import type { Loader, LoaderContext } from 'astro/loaders';
+import { DataEntry, z } from 'astro:content';
+import type { ZodObject, ZodSchema } from 'astro:schema';
+
+interface Link {
+    href: string;
+    description: string;
+}
+
+/**
+* Extract links from "Scontrino" content
+*
+* This parser decorates the default feed object, coming from the feed loader, with the list of links in the "scontrino".
+* It allows to reuse them across the project without errors.
+*/
+function parseLinks(episode: any): Link[] {
+    const links: Link[] = [];
+
+    const SCONTRINO_SEPARATOR = "+++ Scontrino +++";
+
+    let scontrino: string;
```

Comment:
could use a ternary in here?

Reply:
I do not really like if-elses, I like early returns and functions, maybe could also add some tests? bun has a nice built in test runner